pkg/storage/storage_provisioning.go

```diff
@@ -97,16 +102,16 @@ func getValue(obj map[string]interface{}, path ...string) string {
 
 func equal(required *ProvisionedBucket, existing *unstructured.Unstructured) bool {
 	obj := existing.UnstructuredContent()
-	if required.Name != getValue(obj, "spec", "local", "bucket") {
+	if required.Name != getValue(obj, specLiteral, localLiteral, "bucket") {
```

Comment:
"spec" and "local" are replaced with constants but no complaints for "bucket"?

Reply:
The complaints are only if there are more than 4 instances of the string.
"bucket" has only 2 instances.

Reply:
@Raviv-S even if the linter doesn't complain about `bucket ` and it has only 2 instances, I'd define a constant for it too.